src/native/chunk/splitter.cpp

```diff
@@ -44,8 +44,19 @@ Splitter::Splitter(
     if (_calc_md5) {
         extern bool fips_mode;
         if (fips_mode) {
-            posix_memalign((void**)&_md5_mb_mgr, 16, sizeof(MD5_HASH_CTX_MGR));
-            posix_memalign((void**)&_md5_mb_ctx, 16, sizeof(MD5_HASH_CTX));
+            int result;
+            result = posix_memalign((void**)&_md5_mb_mgr, 16, sizeof(MD5_HASH_CTX_MGR));
+            if (result == 0) {
+                // memory assigned successfully
+            } else {
+                // memory not assigned successfully, need to handle the case
+            }
```

Comment:
Do we intentionally do an empty if...else?

Reply:
Thanks, applied some other fixes.